notebooks/create_similarity_matrix.py

```diff
@@ -0,0 +1,366 @@
+import os
+import pathos.multiprocessing
```

Comment:
This is not a system library, so should normally be included in the second block of imports.

Reply:
Okay. Did not know that the order matters that much

Reply:
It doesn't really. But it's a good coding practice to have 3 separated import blocks:
1. System packages.
2. External packages.
3. Local imports.